app/assets/javascripts/discourse/app/components/sidebar/section-link.gjs

```diff
@@ -101,9 +113,22 @@ export default class SectionLink extends Component {
     }
   }
 
+  @bind
+  scrollIntoView(element) {
```

Comment:
```suggestion
  maybeScrollIntoView(element) {
```
might be a better description of what it's doing?

Reply:
Done